### PR TITLE
add lab 3 code

### PR DESCRIPTION
**Pull request by Orest0501**
You need to create text that should be displayed next to the likes. Implement a function like [String]> String, which should use an input array containing the names of people who like the photo

### CODE REVIEW

**Review by Olegbdnr — APPROVED** [review]

File: lab3.cpp [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+
+void like(char* names, char* result)
+{
+    int m = 1, i = 0;
+    char result1[200] = "";
+    int array[15] = { 0 };
```
Comment (zveres-ss) [inline lab3.cpp:11]: make 15 and 2 as constants

File: lab3.cpp [hunk 2]
```diff
@@ -0,0 +1,99 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+
+void like(char* names, char* result)
+{
+    int m = 1, i = 0;
+    char result1[200] = "";
```
Comment (zveres-ss) [inline lab3.cpp:10]: 200 should be declared either as constant or #define

File: lab3.cpp [hunk 3]
```diff
@@ -0,0 +1,99 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+
+void like(char* names, char* result)
+{
+    int m = 1, i = 0;
```
Comment (zveres-ss) [inline lab3.cpp:9]: what m variable name means?

File: lab3.cpp [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    }
+
+    else if (m == 3)
+    {
+        strcat(result, ",");
+        for (int k = array[1]; k < array[2]; ++k)
+        {
+            sym[0] = names[k];
+            strcat(result, sym);
+        }
```
Comment (zveres-ss) [inline lab3.cpp:61]: same code as in lines 46-50

**Review by zveres-ss — CHANGES_REQUESTED** [review]

**Pull request merged**